npm/design-system/src/core/button/Button.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react'
+import { action } from '@storybook/addon-actions'
+
+import { createStory, createStorybookConfig } from '../../stories/util'
+
+import { Button as ButtonComponent, LinkButton } from './Button'
+import { IconButton as IconButtonComponent } from './IconButton'
+
+import typography from '../../css/derived/jsTypography.scss'
+import { TextSize } from '../../css'
+import { PaddedBox } from '../surface/paddedBox/PaddedBox'
+import { Icon } from '../icon/Icon'
+
+export default createStorybookConfig({
+  title: 'Core/Button',
+})
+
+export const Button = createStory(() => (
+  <div>
+    <PaddedBox>
+      <ButtonComponent aria-label="buttonPress" onPress={action('buttonPress')}>Simple button</ButtonComponent>
+      <LinkButton aria-label="anchorButtonPress" onPress={action('anchorButtonPress')}>Anchor button</LinkButton>
+    </PaddedBox>
+    <PaddedBox style={{ backgroundColor: 'var(--brand-00)' }}>
+      <ButtonComponent aria-label="buttonPress" color='white' onPress={action('buttonPress')}>Simple button</ButtonComponent>
+      <LinkButton aria-label="anchorButtonPress" color='white' onPress={action('anchorButtonPress')}>Anchor button</LinkButton>
+    </PaddedBox>
+    <PaddedBox>
+      <ButtonComponent aria-label="buttonPress" color='white' onPress={action('buttonPress')}>Simple button</ButtonComponent>
+      <LinkButton aria-label="anchorButtonPress" color='white' onPress={action('anchorButtonPress')}>Anchor button</LinkButton>
+    </PaddedBox>
+  </div>
+))
+
+export const ButtonSizes = createStory(() => (
+  <div>
+    <div style={{ width: 500 }}>
+      {Object.keys(typography).filter((key) => key !== 'type' && !key.startsWith('line-height') && !key.startsWith('text-mono')).map((key) => {
```

Comment:
Nit: we could use `reduce` here to have a lot less loops

Reply:
I would be interested in seeing what you suggest. This is a story though, so I don't think we're very concerned with performance.

Reply:
Completely untested, might have the condition backwards, but something like:

```ts
{typography.reduce((acc, key) => {
  if (key !== 'type' && !key.startsWith('line-height') && !key.startsWith('text-mono')) {
    return acc
  }

  const size = key.replace('text-', '')
 
  return acc.concat(
    return (
      <ButtonComponent
        key={key}
        size={size as TextSize}
        aria-label="buttonPress"
      >
         `Button ${size}`}
      </ButtonComponent>
    )
  )
},[])}
```

Basically whenever I see filter + map I assume they can be combined into reduce. That said, I normally do this for perf and I don't see any real problem either way.